website/docs/reference/global-configs.md

```diff
@@ -248,6 +283,11 @@ config:
   send_anonymous_usage_stats: False
 ```
 
+You can also use the DO_NOT_TRACK environmental variable to enable or disable sending anonymous data. For more information, see [Environmental variables](/dbt-cloud/using-dbt-cloud/cloud-environment-variables).
+
+`DO_NOT_TRACK=1` is the same as `DBT_SEND_ANONYMOUS_USAGE_STATS=False`
+`DO_NOT_TRACK=2` is the same as `DBT_SEND_ANONYMOUS_USAGE_STATS=True`
```

Comment:
Sorry, I may have confused this one. The two are both boolean and exactly inverses; `DO_NOT_TRACK` just uses `0` instead of `False` by convention.
```suggestion
`DO_NOT_TRACK=0` is the same as `DBT_SEND_ANONYMOUS_USAGE_STATS=True`
```